mini_programs/guess_number.py

```diff
@@ -4,13 +4,13 @@
 
 print('###     Choose Level     ###')
 print('### Easy | Medium | Hard ###')
-level = input('Enter level: ')
+level = input('Enter level: ').lower()
```

Comment:
It’s better to add .lower() here to avoid repetition in `if.. elif..` statement

Reply:
Oh yaa.. you're right. Thanks :)